flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -103,15 +112,17 @@ def transform(input: Union[Channel, List[Channel]],
               output_num=1,
               name: Text = None) -> Union[Channel, Tuple[Channel]]:
     """
-    Transformer operator. Transform the dataset so that the original dataset can be used for trainer or other operators
-    after feature engineering, data cleaning or some other data transformation.
-
-    :param input: List of input data. It contains multiple channels from the operators which generate data.
-    :param transform_processor: The user defined function in transform operator. User can write their own logic here.
-    :param output_num: The output number of the operator. The default value is 1.
-    :param name: Name of the transform operator.
-    :return: Channel or Tuple[Channel]. It returns Channel When the output_num is 1 and returns Tuple[Channel] when
-             the output_num is bigger than 1, and the len(Tuple(Channel)) is output_num.
+    Transforms dataset through the user-defined processor based on the input :class:`~ai_flow.graph.channel.Channel`.
+    The processor generates the transforming :class:`~ai_flow.meta.ai_graph.ai_node.AINode`.
```

Comment:
```
:class:`~ai_flow.meta.ai_graph.ai_node.AINode` -> :class:`~ai_flow.ai_graph.ai_node.AINode`

```

There are some wrong usage in the file as well. Maybe we can do a global replacement to fix the issue.